src/sentry/api/endpoints/organization_missing_org_members.py

```diff
@@ -93,9 +95,9 @@ def _get_missing_organization_members(
             select id
             from sentry_repository
             where provider = %(provider)s
-            and organization_id = %(org_id)s
-            and integration_id in (%(integration_ids)s)
-            )
+            and organization_id = %(org_id)s"""
+    query += integration_query
+    query += """
```

Comment:
Instead of adding multiple strings together, could you use a multiline f-string for readability so the whole query stays together

Reply:
i had followed a suggestion before to not make the entire string an f string. i'm not sure if it's any different from making parts of it f string

Reply:
Do you mean they recommended using the % with passing a dict into `CommitAuthor.objects.raw(query, param_dict)`?

Reply:
yes

Reply:
but doing %s causes all of the values passed into the query dict to be enclosed in parentheses when rendered

Reply:
Is that fine for the query. It seemed like the problem wasn't parenthesis but the extra quotes around the comma separated ids

Reply:
sorry i meant quotations

Reply:
it's something to do with https://docs.djangoproject.com/en/4.2/topics/security/#sql-injection-protection, but since i'm already using f-string i'm not sure what the difference is

Reply:
jk i got it to work just by casting `integration_ids` to a tuple

Reply:
nice, should the tests also be reverted back?

Reply:
no the tests are correct. they test the case that led to the error

Reply:
aka my comment above still stands